_15thnight/queue.py

```diff
@@ -35,10 +34,25 @@ def queue_send_message(email, number, subject, body):
     """
     if number:
         send_sms(to_number=number, body=body)
-    #send_email(email, subject, body)
     message = Message(body=body, subject=subject, recipients=[email])
     mailer.send(message)
 
+
 @celery.task
 def queue_send_email(message):
-    mailer.send(message)
\ No newline at end of file
+    mailer.send(message)
+
+
+@celery.task
+def queue_send_alert_resolution_notice(need):
+    # Check if alert is closed and trigger an alert closed notification
+    if not need.alert.is_open():
+        alert = need.alert
+        body = "Alert {0} has been closed. See {1} for details" % (
+            alert, alert.url())
+        providers = [provider for provider in alert.providers_notified]
+        subject = "Alert %s has closed." % alert.id
+
+        for provider in providers:
+            queue_send_message(
+                provider.email, provider.phone_number, subject, body)
```

Comment:
To avoid complexity, `queue` tasks should not be firing off other `queue` tasks. Functions in `core` fire off all queue tasks.